rsocket/framing/FrameTransport.cpp

```diff
@@ -164,8 +143,6 @@ void FrameTransport::cancel() {
 }
 
 void FrameTransport::outputFrameOrDrop(std::unique_ptr<folly::IOBuf> frame) {
```

Comment:
Agree with you. boolean as a return value would be good. However this class is going away so the caller will have to implement the logic if it needs to.